tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -145,8 +201,75 @@ def test_positive_validate_katello_certs_check_output(self, cert_setup, cert_dat
             )
         self.validate_output(result, cert_data)
 
+    @pytest.mark.tier1
+    def test_katello_certs_check_output_wildcard_inputs(
+        self, generate_certs, cert_data, certs_cleanup
+    ):
+        """Validate that katello-certs-check generates correct output with wildcard certs.
+
+        :id: 7f9da806-5b23-11eb-b7ea-d46d6dd3b5b2
+
+        :steps:
+
+            1. Get valid wildcard certs from generate_certs
+            2. Run katello-certs-check with the required valid arguments
+               katello-certs-check -c CERT_FILE -k KEY_FILE -r REQ_FILE
+               -b CA_BUNDLE_FILE
+            3. Assert the output has correct commands with options
+
+        :expectedresults: katello-certs-check should generate correct commands
+         with options.
+        """
+        with get_connection() as connection:
+            result = connection.run(
+                'katello-certs-check -c {} -k {} -b {}'.format(
+                    "certs/wildcard.crt",
```

Comment:
Did this change? Why is .format being used to inject hard-coded strings?  Could have sworn in prior commit these were variables.